src/extends/room/construction.js

```diff
@@ -30,7 +30,6 @@ const structureMap = [
 
 const structures = Object.keys(CONTROLLER_STRUCTURES)
 const skipStructures = [
-  STRUCTURE_ROAD,
```

Comment:
This could cause issues with the practical room level calculations, and we may want to treat room construction separately because this system is really dependent on room level controls (which doesn't apply to roads, ramparts, or walls in the same way they do other things).

Reply:
Ah, I wasn't sure what this did. Hehe. Guess I was wrong. So it builds roads even if they are in this list?

Reply:
Sure looks like it doesn't build roads while it's in this list...

Reply:
So yeah, agreed on treating construction separately from the room level calculation, cause skipping roads, ramparts etc just won't do.

Reply:
Removing this (or adding it) should have no effect on construction itself. However, it could make the `practicalRoomLevel` function return something weird, which in turn could prevent other structures from getting built.

Reply:
Well, it is used by getNextMissingStructureType, which is used by constructNextMissingStructure...